cmd/spire-agent/cli/run/run.go

```diff
@@ -70,6 +70,8 @@ type agentConfig struct {
 	TrustDomain                   string    `hcl:"trust_domain"`
 	AllowUnauthenticatedVerifiers bool      `hcl:"allow_unauthenticated_verifiers"`
 
+	AuthorizedUsersDelegatedIdentityApi []string `hcl:"authorized_users_delegated_identity_api"`
```

Comment:
Wonder if we can shorten this name....
```suggestion
	AuthorizedDelegatees []string `hcl:"authorized_delegatees"`
```

Reply:
Yeah, `authorized_users_delegated_identity_api` is to big I think... is a good idea to short the name... there are some others place in code, maybe we can do the same